code/modules/examine/examine.dm

```diff
@@ -1,62 +1,54 @@
-/*	This code is responsible for the examine tab.  When someone examines something, it copies the examined object's description_info,
-	description_fluff, and description_antag, and shows it in a new tab.
-
-	In this file, some atom and mob stuff is defined here.  It is defined here instead of in the normal files, to keep the whole system self-contained.
-	This means that this file can be unchecked, along with the other examine files, and can be removed entirely with no effort.
-*/
-
-
-/atom/
-	var/description_info = null //Helpful blue text.
-	var/description_fluff = null //Green text about the atom's fluff, if any exists.
-	var/description_antag = null //Malicious red text, for the antags.
-
-//Override these if you need special behaviour for a specific type.
-/atom/proc/get_description_info()
-	if(description_info)
-		return description_info
-	return
-
-/atom/proc/get_description_fluff()
-	if(description_fluff)
-		return description_fluff
-	return
-
-/atom/proc/get_description_antag()
-	if(description_antag)
-		return description_antag
-	return
-
-/mob/living/get_description_fluff()
+/**
+ * Used for showing a more detailed description in the 'Examine' tab after examining the atom.
+ *
+ * Shown as blue text in the Examine tab.
+ */
+/atom/proc/detailed_examine()
+	return ""
```

Comment:
Ok I am gonna say it.

Can you make this take `mob/user` as an arg, so we can tweak this text from the user, similar to `examine(mob/user)`


Also `return null` for these

Reply:
What would be the use of having the user as an argument? I could see in the future maybe having different text for different mob types, but currently it's all set up to only have a single description.
<hr>

> Also `return null` for these

Yep 👍
I was trying to imply that it needs to return something, but `return null` works just as well for that.